data-raw/R2R_TMH_habitat_inputs/cache_tmh_dataR.R

```diff
@@ -0,0 +1,306 @@
+library(tidyverse)
+
+# file to cache Theoretical Max Habitat data objects 
+# and some exploratory plots to compare SIT existing 
+# to TMH
+
+all_habitat_data <- readRDS('data-raw/R2R_TMH_habitat_inputs/all_habitat_data_for_tmh_inputs_fall_run.rdata')
```

Comment:
Is all habitat data existing habitat data or tmh data? Name is a little confusing for me

Reply:
changed the name